.github/workflows/e2e.yml

```
@@ -0,0 +1,118 @@
+name: e2e
+
+on:
+  push:
+    branches: [main]
+  workflow_dispatch:
+
+
+env:
+  LAUNCHABLE_DEBUG: 1
+  LAUNCHABLE_REPORT_ERROR: 1
+
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.5, 3.6, 3.7, 3.8, 3.9]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/checkout@v2
+        with:
+          repository: launchableinc/examples
+          path: examples
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - uses: actions/setup-go@v2
+        with:
+          go-version: 1.17
+      - uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install .
+      # bazel
+      - name: Install Bazelisk
+        run: |
+          go get github.com/bazelbuild/bazelisk
```

Because we use Go 1.17 go install might be better for the binary installation
```suggestion
          go install github.com/bazelbuild/bazelisk
```